fmriprep/workflows/bold.py

```diff
@@ -15,26 +19,30 @@
 import pkg_resources as pkgr
 
 from niworkflows.nipype import logging
+from niworkflows.nipype.utils.filemanip import split_filename
 from niworkflows.nipype.pipeline import engine as pe
 from niworkflows.nipype.interfaces import ants, afni, c3, fsl
 from niworkflows.nipype.interfaces import utility as niu
 from niworkflows.nipype.interfaces import freesurfer as fs
-from niworkflows.interfaces.registration import EstimateReferenceImage
-import niworkflows.data as nid
-
-from niworkflows.interfaces import SimpleBeforeAfter
-from fmriprep.interfaces import DerivativesDataSink, InvertT1w, ValidateImage
 
-from fmriprep.interfaces.images import GenerateSamplingReference, extract_wm
-from fmriprep.interfaces.nilearn import Merge
-from fmriprep.interfaces.reports import FunctionalSummary
-from fmriprep.workflows import confounds
-from niworkflows.nipype.utils.filemanip import split_filename
-from fmriprep.workflows.fieldmap.unwarp import init_pepolar_unwarp_wf
-from fmriprep.workflows.util import (
-    init_enhance_and_skullstrip_epi_wf, init_skullstrip_epi_wf,
+import niworkflows.data as nid
+from niworkflows.interfaces.registration import EstimateReferenceImage
+from niworkflows.interfaces import SimpleBeforeAfter, NormalizeMotionParams
+
+from ..interfaces import (
+    DerivativesDataSink, InvertT1w, ValidateImage, GiftiNameSource, GiftiSetAnatomicalStructure
+)
+from ..interfaces.images import GenerateSamplingReference, extract_wm
+from ..interfaces.nilearn import Merge
+from ..interfaces.reports import FunctionalSummary
+from ..workflows import confounds
+from ..workflows.fieldmap.unwarp import init_pepolar_unwarp_wf
+from ..workflows.util import (
+    init_enhance_and_skullstrip_bold_wf, init_skullstrip_bold_wf,
     init_bbreg_wf, init_fsl_bbr_wf)
 
+
+DEFAULT_MEMORY_MIN_GB = 0.01
```

Comment:
Not for this PR, but this makes me wonder if we can make this a config option in nipype. It would be nice to set an app-level default.